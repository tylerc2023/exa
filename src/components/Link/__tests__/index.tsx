import * as React from "react";
import { create } from "react-test-renderer";
import { Link, Router } from "../..";

it("renders correctly", () => {
  const dom = create(
    <Router>
      <Link title="hello" to="/" />
    </Router>
  ).toJSON();
  expect(dom).toMatchSnapshot();
});

it("renders correctly without defaults", () => {
  const dom = create(
    <Router>
      <Link title="hello" to="/" underlayColor="blue" />
    </Router>
  ).toJSON();
  expect(dom).toMatchSnapshot();
});
