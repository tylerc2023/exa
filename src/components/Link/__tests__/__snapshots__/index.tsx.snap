// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={null}
  >
    <Text
      style={
        Array [
          Object {
            "fontSize": 14,
            "fontWeight": "600",
            "letterSpacing": 0.75,
          },
          Object {
            "textAlign": "center",
          },
          undefined,
          null,
        ]
      }
    >
      HELLO
    </Text>
  </View>
</View>
`;

exports[`renders correctly without defaults 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={null}
  >
    <Text
      style={
        Array [
          Object {
            "fontSize": 14,
            "fontWeight": "600",
            "letterSpacing": 0.75,
          },
          Object {
            "textAlign": "center",
          },
          undefined,
          null,
        ]
      }
    >
      HELLO
    </Text>
  </View>
</View>
`;
